frb_example/pure_dart/rust/src/api.rs

```diff
@@ -1204,3 +1204,18 @@ pub fn return_non_dropable_dart_opaque(opaque: DartOpaque) -> SyncReturn<DartOpa
 pub fn frb_generator_test() -> RustOpaque<FrbOpaqueReturn> {
     panic!("dummy code");
 }
+
+pub fn handle_type_alias_id(input: Id) -> Id {
```

Comment:
nit: Does this PR support more advanced use cases, or it only supports primitive types (e.g. u64)? If so, we should test them to show the powerfulness of the new feature

Reply:
Do you mean I need to add tests for i32 or other primitive types?

the main purpose is to support the use of ItemType in the Schema(like `TestModel`)

Reply:
Currently, when reading code, people will think "oh I know it supports case like TestModel where Id==int". But if it supports many more, e.g. `type Sth = MyFancyStruct` or `type Sth = MyFancyEnum` then would be great to show it